common/api/core-common.api.md

```diff
@@ -10105,13 +10147,15 @@ export class TextStyleSettings {
     readonly color: TextStyleColor;
     static defaultProps: Readonly<Required<TextStyleSettingsProps>>;
     static defaults: TextStyleSettings;
+    readonly elbowLength: number;
```

Comment:
I think it would be helpful to define an interface type that contains all of the leader-related settings, and give TextStyleSettings a single property of that type. Mixing them all in as top-level properties makes it very difficult to tell which are related to leaders, difficult to write APIs that operate on the leader-specific properties as a unit, and make conflicts/confusion likely in future (for example, if we add new annotations for which elbows are relevant).